resources/install.rb

```diff
@@ -21,26 +20,34 @@
 property :security_mode, String, equal_to: ['Windows Authentication', 'Mixed Mode Authentication'], default: 'Windows Authentication'
 property :sa_password, String
 property :sysadmins, [Array, String], default: ['Administrator']
-property :agent_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :agent_startup, String, equal_to: ['Automatic', 'Manual', 'Disabled', 'Automatic (Delayed Start)'], default: 'Disabled'
+property :agent_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :agent_account_pwd, String
+property :ft_account, String, default: 'NT Service\MSSQLFDLauncher'
+property :ft_account_pwd, String
 property :rs_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :rs_account_pwd, String
 property :rs_startup, String, equal_to: ['Automatic', 'Manual', 'Disabled', 'Automatic (Delayed Start)'], default: 'Automatic'
 property :rs_mode, String, default: 'FilesOnlyMode'
 property :as_sysadmins, [Array, String], default: ['Administrator']
 property :sql_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :sql_account_pwd, String
+property :sql_instant_file_int, [true, false], default: false
+property :is_account, String, default: 'NT AUTHORITY\NetworkService'
+property :is_account_pwd, String
+property :is_startup, String, default: 'Automatic'
```

Comment:
Can you add the equal_to: for valid actions this prevents an issue if the user makes a typo.